app/src/main/java/com/example/felipe/criminalintent/CrimeAdapter.kt

```diff
@@ -21,7 +21,13 @@ class CrimeAdapter(crimes: List<Crime>, private val callback: (Crime) -> Unit) :
 
     override fun getItemCount() = crimes.size
 
-    fun notifyItemChangedByID(index: UUID) = notifyItemChanged(crimes.indexOfFirst { it.id == index })
+    fun notifyItemChangedByID(ids: ArrayList<String>) {
```

Comment:
add `@MainThread`

Reply:
Always reference the interfaces (List in this case)